experiment/autobumper/bumper/bumper.go

```diff
@@ -229,6 +229,16 @@ func cdToRootDir() error {
 }
 
 func Call(stdout, stderr io.Writer, cmd string, args ...string) error {
+	(&logrus.Logger{
+		Out:       stderr,
+		Formatter: logrus.StandardLogger().Formatter,
```

Comment:
Will this carry forward the secretAgent/censor?

Reply:
yeah, that's why it uses the passed-in stderr rather than `os.stderr`